big_tests/tests/inbox_helper.erl

```diff
@@ -41,6 +41,7 @@
          leave_room/3,
          make_members/3,
          mark_last_muclight_message/2,
+         mark_last_muclight_message/3,
```

Comment:
Do we need this export?

Reply:
Ah, I forgot to change one thing, there's a reason why this export makes the test a bit clearer. In `non_reset_marker_should_not_affect_muclight_inbox`, the part where the conversation is marked wrong, can be done in one line using this function. I'll push it.